drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/main/java/org/drools/workbench/screens/guided/dtable/backend/server/conversion/SubHeaderBuilder.java

```diff
@@ -64,14 +83,14 @@ public void build() {
 
         final List<BaseColumn> expandedColumns = dtable.getExpandedColumns();
 
-        for (int sourceIndex = 0; sourceIndex < expandedColumns.size(); sourceIndex++) {
+        for (; sourceColumnIndex < expandedColumns.size(); sourceColumnIndex++) {
```

Comment:
I am reading the class again again and I think we do not need `sourceColumnIndex` to be class variable. I think we can keep its scope just in the `for` loop. Then I think the only usage of `sourceColumnIndex` can be replaced by `columnIndex`.